### DISCUSSION TITLE
Which North is used? Magnetic North or True North?

### DISCUSSION THREAD

**Discussion by sequoia-hope** [geopy/geopy#507] [Help] [answered]
Hello! I have been asked to align something with True North. I noticed that the geopy documentation does not say whether bearing is referenced against magnetic north or true north, and wikipedia says absolute bearing can be referenced by either of those.

I'm not really sure how to determine this so I thought I would ask. Thank you!

**Comment by cffk** [comment, marked as answer]
It's true north.

**Reply by sequoia-hope** [reply]
Okay thank you.
